app/views/projects/blob/_replace.html.haml

```diff
@@ -0,0 +1,28 @@
+#modal-replace-blob.modal
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %a.close{href: "#", "data-dismiss" => "modal"} ×
+        %h3.page-title Replace
```

Comment:
sorry, It should have, I think I removed during change between upload and replace.